Cargo.toml

```diff
@@ -26,9 +26,9 @@ anyhow = "1"
 
 serde_json = "1"
 
-skar-client = "0.5.2"
-skar-net-types = "0.1.1"
-skar-format = "0.2.0"
+skar-client = "0.6"
+skar-net-types = "0.1"
```

Comment:
A downgrade?

Reply:
The patch version doesn't matter if there is a minor version because it implicitly converts it into "^x.x.x"